backend/inspirehep/orcid/tasks.py

```diff
@@ -268,3 +269,13 @@ def _find_user_matching(orcid, email):
     if user_identity:
         return user_identity.user
     return User.query.filter_by(email=email).one_or_none()
+
+
+@shared_task(ignore_result=False, max_retries=3)
```

Comment:
with ``ignore_results=False`` the endpoint will wait to schedule all the tasks first

Reply:
yeah copy-paste, needs `True` indeed.